library/std/src/env.rs

```diff
@@ -219,18 +215,18 @@ fn _var(key: &OsStr) -> Result<String, VarError> {
 }
 
 /// Fetches the environment variable `key` from the current process, returning
-/// [`None`] if the variable isn't set.
-///
-/// # Panics
-///
-/// This function may panic if `key` is empty, contains an ASCII equals sign
-/// `'='` or the NUL character `'\0'`, or when the value contains the NUL
-/// character.
+/// [`None`] if the variable isn't set or there's another error.
 ///
 /// Note that the method will not check if the environment variable
 /// is valid Unicode. If you want to have an error on invalid UTF-8,
 /// use the [`var`] function instead.
 ///
+/// # Errors
+///
+/// Errors if the variable isn't set.
+/// May error if the `key` is empty, contains an ASCII equals sign `'='`, or contains the NUL character `'\0'`.
```

Comment:
Btw, I think lowercase "null" is now the preferred term? See #84842.

Reply:
I think that's for null, as in the pointer. This is for the character. I could type it nul, but I would think we would want to match EOF (which is admittedly a pseudocharacter) and DEL, which I usually see in upper case?

Reply:
Fair enough!